src/components/APIStarter.jsx

```diff
@@ -1,18 +1,373 @@
+  //         answer_d_correct: "false",
+  //         answer_e_correct: "false",
+  //         answer_f_correct: "false",
+  //       },
+  //       correct_answer: "answer_a",
+  //       explanation: null,
+  //       tip: null,
+  //       tags: [
+  //         {
+  //           name: "PHP",
+  //         },
+  //       ],
+  //       category: "Code",
+  //       difficulty: "Medium",
+  //     },
+  //     {
+  //       id: 924,
+  //       question: "Which of the following is not a services in Kubernetes?",
+  //       description: null,
+  //       answers: {
+  //         answer_a: "Cluster IP",
+  //         answer_b: "Node port",
+  //         answer_c: "Load Balancer",
+  //         answer_d: "External name",
+  //         answer_e: "Container Proxy",
+  //         answer_f: null,
+  //       },
+  //       multiple_correct_answers: "false",
+  //       correct_answers: {
+  //         answer_a_correct: "false",
+  //         answer_b_correct: "false",
+  //         answer_c_correct: "false",
+  //         answer_d_correct: "false",
+  //         answer_e_correct: "true",
+  //         answer_f_correct: "false",
+  //       },
+  //       correct_answer: null,
+  //       explanation: null,
+  //       tip: null,
+  //       tags: [
+  //         {
+  //           name: "Kubernetes",
+  //         },
+  //       ],
+  //       category: "DevOps",
+  //       difficulty: "Easy",
+  //     },
+  //     {
+  //       id: 676,
+  //       question: "What is the login name of the administrator in Linux?",
+  //       description: null,
+  //       answers: {
+  //         answer_a: "root",
+  //         answer_b: "administrator",
+  //         answer_c: "admin",
+  //         answer_d: "All of the mentioned",
+  //         answer_e: "None of the mentioned",
+  //         answer_f: null,
+  //       },
+  //       multiple_correct_answers: "false",
+  //       correct_answers: {
+  //         answer_a_correct: "true",
+  //         answer_b_correct: "false",
+  //         answer_c_correct: "false",
+  //         answer_d_correct: "false",
+  //         answer_e_correct: "false",
+  //         answer_f_correct: "false",
+  //       },
+  //       correct_answer: null,
+  //       explanation: null,
+  //       tip: null,
+  //       tags: [
+  //         {
+  //           name: "BASH",
+  //         },
+  //         {
+  //           name: "Linux",
+  //         },
+  //       ],
+  //       category: "Linux",
+  //       difficulty: "Easy",
+  //     },
+  //   ]);
 
   const [index, setIndex] = useState(0);
 
+  // useEffect(() => {
+  //   fetch(`${BASEURL}?apiKey=${process.env.REACT_APP_API_KEY}&limit=10`)
+  //     .then((response) => response.json())
+  //     .then((json) => setResponse(json))
+  //     .catch((error)=> console.log(error));
+  //   console.log("Sometinhg")
+  // }, []);
+
+  const storeCorrectAns = (response) => {
+    let ans = {};
+    Object.entries(response).forEach(([key, value]) => {
```

Comment:
Change Object.entries() to Object.values() because you are not using `key` inside the logic.

Reply:
Done Tutor!